pkg/database/db.go

```diff
@@ -1,15 +1,40 @@
 package database
 
 import (
-	"database/sql"
-	// "github.com/go-sql-driver/mysql"
+	"errors"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"log"
+	"os"
 )
 
-// Init initializes an instance of the database
-func Init() (*sql.DB, error) {
-	db, err := sql.Open("mysql", "root@cloudsql(project-id:instance-name)/dbname")
+// MustInit initializes an instance of the database
+func MustInit() *gorm.DB {
```

Comment:
Yeah i remember Josh saying that func Init shouldn't return anything and I was about to ask you lol

Reply:
Changed in 64452d2c75d13566f91b0608d34ae9873426a337